app/views/receipts/_form.html.erb

```diff
@@ -241,43 +269,52 @@
         </div>
 
         <div class="actions">
-          <%= f.submit :class => 'btn btn-primary', :id => 'submit-receipt' %>
+          <% if !@edit %>
+            <%= f.submit "Create and add Another", :class => 'btn btn-primary', :id => 'submit-receipt-and-new',:name => 'submit_receipt_and_new'%> 
+          <%end%>
+            <%= f.submit :class => 'btn btn-primary', :id => 'submit-receipt', :name => 'submit_receipt'  %>
         </div>
+
+         
+
       </div>
     </div>
 
   </div>
   <% end %>
 
   <script type="text/javascript">
```

Comment:
Can we move script below this line to corresponding *.js file? This view is very cluttered and difficult to navigate through.